tasks/php-fpm.yml

```diff
@@ -17,3 +17,7 @@
   lineinfile: dest="{{php_config_prefix}}/fpm/php.ini"
               regexp=';?opcache.enable=\d'
               line='opcache.enable=1'
+- name: disable expose_php
+  lineinfile: dest=/etc/php5/fpm/php.ini
```

Comment:
@ricardofontanelli could you please use the config place holder `{{php_config_prefix}}` here instead of the absolute path?